### PR TITLE
CorrectifPagePhilo

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/TextPhilosophy/index.jsx [hunk 1]
```diff
@@ -32,18 +32,15 @@ export default function TextPhilosophy() {
       <Title title="Editos" />
       <article className="left">
         <div className="image">
-          {players
-            .filter((player) => player.id === 5)
-            .map((player) => {
+          {staff
+            .filter((person) => [21].includes(person.role_id))
```
Comment (LoicBrassart) [inline frontend/src/components/TextPhilosophy/index.jsx:36]:
```
.filter((person) => person.role_id===21)
```

Comment (LoicBrassart) [reply frontend/src/components/TextPhilosophy/index.jsx:36]: (Adaptable for all subsequent codes, of course)

**Review by LoicBrassart — APPROVED** [review]
Strange bit of code, but it should work

**Pull request merged**